File: Septembar2K23/WebTemplate/Models/Sastojak.cs
using System.Text.Json.Serialization;

namespace WebTemplate.Models
{
    public class Sastojak
    {
        [Key]
        public int ID { get; set; }
        public required string Naziv { get; set; }
        public required int Kolicina { get; set; }
        public int Cena { get; set; }
        [JsonIgnore]
        public Zarada? Zarade { get; set; }

    }
}

File: Septembar2K23/WebTemplate/Client/Sastojak.js
export class Sastojak {
  constructor(id, naziv, cena, kolicina) {
    (this.id = id),
      (this.naziv = naziv),
      (this.cena = cena),
      (this.kolicina = kolicina),
      (this.r = Math.floor(Math.random() * 256)),
      (this.g = Math.floor(Math.random() * 256)),
      (this.b = Math.floor(Math.random() * 256));
  }
}
